syntax = "proto3";


message JobBoard {
  // Title of this job board.
  string title = 1;

  // Link to the job board.
  string link = 2;

  // A set of filter IDs: do not show the job board if one these
  // filters returns false for the project.
  // See the list of available filters in scoring.py.
  repeated string filters = 3;
}

message JobBoards {
  // A sorted list of job boards. The ones that Bayes think are best suited are
  // first.
  repeated JobBoard job_boards = 1;
}
